Source/Core/VideoCommon/Assets/MeshAsset.cpp

```diff
@@ -7,7 +7,10 @@
 #include <array>
 #include <utility>
 
-#include <tinygltf/tiny_gltf.h>
+#define TINYGLTF_IMPLEMENTATION
```

Comment:
Why were these defines added?

Reply:
I'm aware of what the documentation says.  But that is not how we are using it.  Please don't change the implementation, in particular when you aren't even able to test.

We should match the existing behavior.  Not sure why we aren't seeing those failures on our normal build.  We might be able to add `TINYGLTF_NO_EXTERNAL_IMAGE` but I'd have to test that.